src/core/errorHandler.js

```diff
@@ -59,13 +59,8 @@ export class ErrorHandler {
 export type ErrorHandlerType = typeof ErrorHandler;
 
 /*
- * This is a decorator that gives a component the ability to handle errors.
- *
- * The decorator will assign an ErrorHandler instance to the errorHandler
- * property.
```

Comment:
Thanks for this patch @willdurand !

Your addition about `withRenderedErrorHandler()` looks right but some of the previous documentation got chopped off. Can you open a new PR to put it back? The doc block should look like this to make it clear that the component will get an `errorHandler` prop:

```js
/*
 * This is a decorator that gives a component the ability to handle errors.
 *
 * The decorator will assign an ErrorHandler instance to the errorHandler
 * property.
 *
 * For convenience, you can use `withRenderedErrorHandler()` which renders the
 * error automatically at the beginning of the component's output.
 *
 * Example ...
 */
```